backend/battle/tests.py

```diff
@@ -1,3 +1,46 @@
-# from django.test import TestCase
+from django.test import TestCase, Client
+from model_bakery import baker
+from users.models import User
+from battle.models import Battle
 
-# Create your tests here.
+'''
+class ListBattlesTest(TestCase):
```

Comment:
When writing tests for things that return a collection, test:
  - Zero
  - One
  - Few
  - A lot
  - Exception
  - [The diff ](https://vinta.atlassian.net/wiki/spaces/development/blog/2021/07/14/2024079365/Testing+the+diff)

Reply:
Alright :)